files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js

```diff
@@ -1190,7 +980,7 @@ class CinnamonMenuApplet extends Applet.TextIconApplet {
     onAppSysChanged() {
         if (this.refreshing == false) {
             this.refreshing = true;
-            Mainloop.timeout_add_seconds(1, Lang.bind(this, this._refreshAll));
+            Mainloop.timeout_add_seconds(1, this._refreshAll.bind(this));
```

Comment:
```suggestion
            Mainloop.timeout_add_seconds(1, () => this._refreshAll());
```

This could also be an arrow function.